File: Programmers/Level2/getDayName/getDayName.py
months = [31,29,31,30,31,30,31,31,30,31,30,31]
weekdays = ["THU","FRI","SAT","SUN","MON","TUE","WED"]

def getDayName(a,b):
    days = b + sum([months[i] for i in range(a-1)])
    return weekdays[days%7]

#아래 코드는 테스트를 위한 출력 코드입니다.
print(getDayName(5,24))

File: Programmers/Level2/getDayName(cjp)/getDayName.cpp
#include<iostream>
#include<string>
using namespace std;

int months[] = {31,29,31,30,31,30,31,31,30,31,30,31};
string weekdays[] = {"THU","FRI","SAT","SUN","MON","TUE","WED"};

string getDayName(int a,int b)
{
  int days = b;
  for(int i=0; i<a-1; ++i)
  {
    days+=months[i];
  }
  return weekdays[days%7];
}

int main()
{
	int a=5,b=24;

	//아래는 테스트 출력을 위한 코드입니다.
	cout<<getDayName(a,b);
}